crates/proof-impl/btc-blockspace/src/prover.rs

```diff
@@ -20,10 +24,17 @@ impl ZkVmProver for BtcBlockspaceProver {
     where
         B: ZkVmInputBuilder<'a>,
     {
+        let block = &input.block;
+
+        let inclusion_proof = match witness_commitment_from_coinbase(&block.txdata[0]) {
+            Some(_) => L1TxProof::generate(&block.txdata, 0),
+            None => L1TxProof::new(0, vec![]),
```

Comment:
The use of `L1TxProof::new(0, vec![])` might not be semantically correct because merkle proof of first txn doesn't necessarily always have zero co-hashes. So perhaps `None` can be used when we don't require inclusion proof for coinbase.